remote-trait-object-tests/src/test_module.rs

```diff
@@ -16,8 +16,8 @@
 
 mod control_loop;
 
-use cbasesandbox::ipc::intra::Intra;
-use cbasesandbox::ipc::{Ipc, IpcRecv, IpcSend};
+use cbasesandbox::transport::intra::Intra;
+use cbasesandbox::transport::{Ipc, TransportRecv, TransportSend};
```

Comment:
Here is `Ipc`

Reply:
I checked all occurrences of `Ipc`. updated.